cmd/edgectl/aes_install.go

```diff
@@ -453,6 +458,18 @@ func (i *Installer) Perform(kcontext string) error {
 		return err
 	}
 
+	versions, err := i.SilentCaptureKubectl("get versions", "", "version", "-o", "json")
+	if err != nil {
+		i.Report("fail_no_cluster")
+		return err
+	}
+	k8sVersion := &kubernetesVersion{}
+	err = json.Unmarshal([]byte(versions), k8sVersion)
+	if err != nil {
+		i.log.Printf("failed to read Kubernetes client and server versions: %v", err.Error())
+	}
+	i.k8sVersion = k8sVersion
```

Comment:
Is a zero-valued structure okay here in the error case? Would `nil` be better? A comment either way would be helpful. Otherwise I'll pause at this bit of code every time I'm looking through the file.